src/helpers/validation/articleValidation.js

```diff
@@ -0,0 +1,23 @@
+// import validator from 'validatorjs';
```

Comment:
#### Critical
Why is the entire file commented out? If it's not needed, why not just delete it?